src/components/app/MenuButtons/Publish.js

```diff
@@ -132,49 +117,30 @@ class MenuButtonsPublishImpl extends React.PureComponent<
         ) : null}
         <form className="menuButtonsPublishContent" onSubmit={attemptToPublish}>
           <div className="menuButtonsPublishIcon" />
+          <h1 className="menuButtonsPublishTitle">Share Performance Profile</h1>
           <p className="menuButtonsPublishInfoDescription">
-            You’re about to publicly share your profile, which can potentially
-            contain personally identifiable information.
+            Upload your profile and make it accessible to anyone with the link.
+            {shouldSanitizeByDefault
+              ? ' By default, your personal data is removed.'
+              : ' This profile is from Firefox Nightly, so by default all information is included.'}
```

Comment:
It can be also a custom build. Should we keep here "Firefox Nightly"?

Reply:
It seemed simpler to just say Firefox Nightly. I feel like it's accurate enough. I believe when you do a custom build it always says Nightly, right?

Reply:
It becomes `NightlyDebug` generally. But yeah, probably it's simpler that way.